internal/spdxlicense/generate_license_list.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
I think you mentioned this line is getting added by accident. You might want to make a local adjustment to avoid accidentally checking this in 🤔

Reply:
Yep - thought I removed this configuration but it seems like it's back with a vengence. I'll look at what magic nonsense exists within my defaults that is appending this.

Reply:
@luhring looks like my tooling is trying to migrate us to the current spec:
https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md

Reply:
Better explanation of the transition
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives

I'm going to remove it for now, but will keep an eye on the transition going forward. I don't know if it's my `LSP` that is doing this or if it's some configuration I updated in my dotfiles when go1.17 dropped which has slipped my mind.